src/SQLQueryStress/App.config

```diff
@@ -3,4 +3,4 @@
   <runtime>
     <gcServer enabled="true"/>
   </runtime>
-<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/></startup></configuration>
+<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2"/></startup></configuration>

```

Comment:
Can we revert this, the tool is most likely used on many systems that do not have  .NET 4.7.2 installed yet

Reply:
Yup, I have reverted this and rebased my fix-warnings branch on my master too.
I committed the change a few minutes ago, but Github has not updated yet?

Reply:
aha, I think I've done it correctly now.